internal/gatewayapi/testdata/clienttrafficpolicy-invalid-settings.in.yaml

```diff
@@ -0,0 +1,147 @@
+clientTrafficPolicies:
```

Comment:
Do we need to add this test file. 

as long as it passes test, I guess that there's no harm...

Reply:
tests pass, lets keep it ?